packages/govern-token/contracts/MerkleDistributor.sol

```diff
@@ -48,7 +49,7 @@ contract MerkleDistributor is Initializable {
 
     function _verifyBalanceOnTree(uint256 _index, address _to, uint256 _amount, bytes32[] memory _proof) internal view returns (bool) {
         bytes32 node = keccak256(abi.encodePacked(_index, _to, _amount));
-        return !MerkleProof.verify(_proof, merkleRoot, node);
+        return MerkleProof.verify(_proof, merkleRoot, node);
```

Comment:
Can you elaborate closer on why inverting isn't required?

Reply:
Let's say that `verify` returns true. If so, `_verifyBalanceOnTree` would return false. Then, if we look at this check:

https://github.com/aragon/govern/blob/55ccf385b28d5d05443d5170b0e463ccc145ffd7/packages/govern-token/contracts/MerkleDistributor.sol#L37 we can see that in the `require`, it would get `false` and the whole transaction would fail even though it should have been succeeded.

Reply:
Thanks for the closer explanation! :)